Python Examples/TestMonitor/test_data_manager_client.py

```diff
@@ -0,0 +1,139 @@
+import sys
+import requests
+
+create_results_host = "nitestmonitor/v2/results"
+create_steps_host = "nitestmonitor/v2/steps"
+update_results_host = "nitestmonitor/v2/update-results"
+update_steps_host = "nitestmonitor/v2/update-steps"
```

Comment:
Renaming all these `host` into `route` will be better

Reply:
applied the change